Sources/XcodeGenCLI/Commands/ProjectCommand.swift

```diff
@@ -18,6 +18,9 @@ class ProjectCommand: Command {
     @Flag("-n", "--no-env", description: "Disable environment variable expansions")
     var disableEnvExpansion: Bool
 
+    @Key("-r", "--project-root", description: "The path to the project root directory. Defaults to the directory containing the project spec.")
```

Comment:
Nit: can you move this to be between `-s` and `-n`?

Reply:
Done.